CHANGELOG.md

```diff
@@ -6,7 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-This package does not yet have a release
+## [0.11.0] - 2021-05-14
+
+### Changed
+
+* Removed `repo_name` parameter of the `get_updated_repo` function to give the user control of exactly where to clone the repo contents to.
```

Comment:
could you include a reference to the fact that the `clone_dir` argument is no longer a parent directory, and is the name of the directory in which a repo will be cloned?